src/main/scala/quality.scala

```diff
@@ -102,19 +101,15 @@ case object Quality {
 
   final
   def scoreFrom(errP: ErrorP): Score =
-    // ((-10) * errP.log(10)).round().intValue
-    Math.round((-10:Double) * errP.log(10)).toInt
-
+    Math.round( -10 * Math.log10(errP) ).toInt
```

Comment:
keep the type ascription; you're actually calling a different method here.

Reply:
Done in 83da952. Although in both cases it's unambigous.